config/database.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+require("dotenv").config({ path: "../.env" });
```

Comment:
Maybe I just don't get it: why do you `destruct` here?

Reply:
I saw it on a tutorial. Since the .env is in the root but the database.js is inside config, I though it was better to specify the path